chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -141,6 +141,30 @@
   source "crowbar.sh"
 end
 
+directory "/etc/sudoers.d" do
+  owner "root"
+  group "root"
+  mode "0750"
+  action :create
+end
+
+ruby_block "Ensure /etc/sudoers.d is included in sudoers" do
+  block do
+    f = Chef::Util::FileEdit.new("/etc/sudoers")
+    f.insert_line_if_no_match(/^#includedir \/etc\/sudoers.d/,
```

Comment:
I guess adding a `$` at the end of the regexp would be a tiny bit more secure.

But other than that, +1